js/apps/admin-ui/src/realm-settings/localization/EffectiveMessageBundles.tsx

```diff
@@ -62,15 +67,21 @@ export const EffectiveMessageBundles = ({
     Partial<EffectiveMessageBundlesSearchForm>
   >({});
   const themes = serverInfo.themes;
-  const themeKeys = themes
-    ? Object.keys(themes).sort((a, b) => a.localeCompare(b))
+  const themeTypes = themes
+    ? localeSort(
+        Object.keys(themes).filter((key) => key !== "common"),
+        (key) => key,
+      )
+    : [];
```

Comment:
Forgot to add this in my original review, but it seems to me that this might be a potentially expensive operation to do on each render. This component likely will re-render a couple of times due to the different types of state in it getting manipulated, so it might be worth it to memoize these values.

```suggestion
  const themeTypes = useMemo(() => {
    if (!themes) {
      return [];
    }
  
    return localeSort(
      Object.keys(themes).filter((key) => key !== "common"),
      (key) => key,
    );
  }, [themes]);
```

Reply:
Thanks @jonkoops you're 100% right!